recipes/swepy/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "swepy" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 91ed36a907399ce7a29d763cf74077ce14d55e8bcdf52b6f1e8b5723f48d322f
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+
+  host:
+    - python
+    - pip
+  run:
+    - matplotlib
+    - python=3.6
```

Comment:
Is this really python 3.6 only?  Also there needs to be space here `python =3.6`

Reply:
@scopatz, it also runs in python 3.7. But I was unsure of how to add both to the meta.yaml file? The NetCDF Operators (NCOpy) are only testing in 3.6/3.7

Reply:
All you need to do here is `python >=3.6`

Reply:
Sorry about that, committed it yesterday but forgot to push it.